Lib/test/test_asyncgen.py

```diff
@@ -1125,6 +1125,27 @@ def make_arange(n):
         res = self.loop.run_until_complete(run())
         self.assertEqual(res, [i * 2 for i in range(1, 10)])
 
+    def test_asyncgen_nonstarted_hooks_are_cancellable(self):
+        messages = []
```

Comment:
Would it be helpful to add a brief comment mentioning the purpose of this test? I don't think the purpose is immediately obvious from the name of the test alone (without the context of the bpo issue and PR comments).

Reply:
I've added a link to the issue, should be enough for digging into the context